crates/bitwarden-json/src/command.rs

```diff
@@ -131,39 +139,70 @@ pub enum ProjectsCommand {
     /// > Requires using an Access Token for login or calling Sync at least once
     /// Retrieve a project by the provided identifier
     ///
-    /// Returns: [ProjectResponse](crate::sdk::response::projects_response::ProjectResponse)
+    /// Returns: [ProjectResponse](bitwarden::secrets_manager::projects::ProjectResponse)
     ///
     Get(ProjectGetRequest),
 
     /// > Requires Authentication
     /// > Requires using an Access Token for login or calling Sync at least once
     /// Creates a new project in the provided organization using the given data
     ///
-    /// Returns: [ProjectResponse](crate::sdk::response::projects_response::ProjectResponse)
+    /// Returns: [ProjectResponse](bitwarden::secrets_manager::projects::ProjectResponse)
     ///
     Create(ProjectCreateRequest),
 
     /// > Requires Authentication
     /// > Requires using an Access Token for login or calling Sync at least once
     /// Lists all projects of the given organization
     ///
-    /// Returns: [ProjectsResponse](crate::sdk::response::projects_response::ProjectsResponse)
+    /// Returns: [ProjectsResponse](bitwarden::secrets_manager::projects::ProjectsResponse)
     ///
     List(ProjectsListRequest),
 
     /// > Requires Authentication
     /// > Requires using an Access Token for login or calling Sync at least once
     /// Updates an existing project with the provided ID using the given data
     ///
-    /// Returns: [ProjectResponse](crate::sdk::response::projects_response::ProjectResponse)
+    /// Returns: [ProjectResponse](bitwarden::secrets_manager::projects::ProjectResponse)
     ///
     Update(ProjectPutRequest),
 
     /// > Requires Authentication
     /// > Requires using an Access Token for login or calling Sync at least once
     /// Deletes all the projects whose IDs match the provided ones
     ///
-    /// Returns: [ProjectsDeleteResponse](crate::sdk::response::projects_response::ProjectsDeleteResponse)
+    /// Returns: [ProjectsDeleteResponse](bitwarden::secrets_manager::projects::ProjectsDeleteResponse)
     ///
     Delete(ProjectsDeleteRequest),
 }
+
+#[cfg(feature = "internal")]
+#[derive(Serialize, Deserialize, JsonSchema, Debug)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub enum FoldersCommand {
+    /// > Requires Authentication
+    /// > Requires an unlocked vault
+    /// Creates a new folder with the provided data
+    ///
+    Create(FolderCreateRequest),
+
+    /// > Requires Authentication
+    /// > Requires an unlocked vault and calling Sync at least once
+    /// Lists all folders in the vault
+    ///
+    /// Returns: [FoldersResponse](bitwarden::platform::folders::FoldersResponse)
+    ///
+    List(EmptyRequest),
```

Comment:
If the enums are empty, they are handled in a different way for the schema generation, from example in the TS side:
```diff
    projects?:         ProjectsCommand;
-   folders?:          FoldersCommand;
+   folders?:          FoldersCommandClass | FoldersCommandEnum;
}

....

+export enum FoldersCommandEnum {
+   List = "list",
+}
```

FoldersCommand gets renamed to FoldersCommandClass and mostly stays the same, but the List action gets added to a separate enum. I feel like this would overcomplicate things for the few cases that don't require inputs